build.sbt

```diff
@@ -59,6 +59,9 @@ lazy val apexls = crossProject(JSPlatform, JVMPlatform)
       "io.github.apex-dev-tools" % "sobject-types"                  % "63.0.0",
       "io.github.apex-dev-tools" % "standard-types"                 % "63.0.0",
       "io.methvin"              %% "directory-watcher-better-files" % "0.18.0",
+      "io.modelcontextprotocol.sdk" % "mcp"                         % "0.10.0",
```

Comment:
These changes look unnecessary, the parent project is not dependent on mcp or using junit